src/sentry/incidents/models.py

```diff
@@ -406,3 +410,55 @@ class TargetType(Enum):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_alertruletriggeraction"
+
+    @property
+    def target(self):
+        if self.target_type == self.TargetType.USER.value:
+            try:
+                return User.objects.get(id=int(self.target_identifier))
```

Comment:
Does the end user provide us with a user.id? Or do they give us a username/email that is then resolved to an id?

Reply:
I need more validation on the api, but however we want them to pass it, I'll resolve it and store the actual id.

Reply:
Could we use the same identifiers that group assignees or adding members to a team uses? It would be nice to avoid another situation like what we have with the global header where it uses the wrong ids.